robot/game_specific.py

```diff
@@ -6,6 +6,9 @@
 WALL = set(range(0, 28))  # 0 - 27
 
 # Currently for SB2018
+GAME_DURATION_SECONDS = 150
+GAME_EXIT_MESSAGE = "Game over!"
```

Comment:
The exit message isn't game specific. This should probably just be baked into the log message in `game.py`

Reply:
The reason for that was to make the function as general as possible, decreasing coupling. Also, we cannot guarantee that this will not change in the future.

Reply:
Agreed, but this file isnt a general constants file, it's for game-specific things, which this message isnt. It's also ok to change things in a string without abstracting it to a variable.

Reply:
@PeterJCLaw I recall you suggested that `GAME_EXIT_MESSAGE` was a constant. Do you remember the justification for this?

Reply:
@trickeydan I believe the justification was to make it as general as possible. (We may want to change it year on year)

Reply:
At the time I was envisioning that the helper was a fully generic alarm mechanism (this was before @prophile mentioned the existence of [`signal.alarm`](https://docs.python.org/3.5/library/signal.html#signal.alarm)).

I don't feel strongly about this constant (though, obviously, the duration does belong in `game_specific.py`).

Reply:
I agree. The duration should be in `game_specific.py`, however I don't think the message should be. Perhaps baking it into the log would be better

Reply:
588762b